encfs/main.cpp

```diff
@@ -621,7 +619,7 @@ int main(int argc, char *argv[]) {
   // encfs_oper.releasedir = encfs_releasedir;
   // encfs_oper.fsyncdir = encfs_fsyncdir;
   encfs_oper.init = encfs_init;
-  encfs_oper.destroy = encfs_destroy;
+  // encfs_oper.destroy = encfs_destroy;
```

Comment:
Unsure about this. Leaving encfs_oper.destroy = NULL does not change the behavoir ob libfuse?

In any case, should be deleted instead of commented out.

Reply:
I verified this, and no, the function is simply called if it exists, but is not replaced by anything else : https://github.com/libfuse/libfuse/blob/fuse_2_9_5/lib/fuse.c#L2614
I just removed the comment.
Thx 👍